backend/src/main/java/com/github/johannesthorbergsson/backend/exceptions/GlobalExceptionHandler.java

```diff
@@ -28,6 +28,13 @@ public ResponseEntity<Map<String, Object>> handleNoSuchOrderException(NoSuchOrde
         responseBody.put(MESSAGE, exception.getMessage());
```

Comment:
I have missed the introduction of this exception handler totally, but I am really impressed by it! It is a very elegant solution and is much better than my verification, validation and authorisation services separately together! And then you just write `.orElseThrow(` and that's it! Magnificent!